.github/workflows/test_PUMS_aggregation.yml

```diff
@@ -0,0 +1,48 @@
+name: Test PUMS Aggregation
+
+on:
+  workflow_dispatch:
+  # push:
+  #   paths:
+  #     - "aggregate/PUMS/**.py"
+  #     - "tests/PUMS/**.py"
+  #     - ".github/workflows/test_PUMS_aggregation.yml"
+  #     - "poetry.lock"
+  #     - "pyproject.toml"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Why does this workflow run on the ubuntu-latest image and not specified to 20.02?

Reply:
This question applies to all the test workflows

Reply:
No good reason, that was the way I set it up. If you want to implement the standardization go for it

Reply:
I made an issue for this. I can open up a feature branch and open a PR into this one

Reply:
Is this change going to be made before merging? @mbh329

Reply:
no